tests/test_maps.py

```diff
@@ -40,177 +16,136 @@ def region2(points):
 ############
 
 
-def test_draw_map():
+def test_draw_map(states):
     """ Tests that draw_map returns HTML """
-    region = MapRegion(geo_path='us-states.json')
-    map_html = draw_map((40, -90), regions=[region], zoom_start=4)
-    assert isinstance(map_html, HTML)
+    states.show()
 
 
 def test_setup_map():
-    """ Tests that to_html() returns HTML """
-    center = [45.5244, -122.6699]
+    """ Tests that passing kwargs doesn't error. """
     kwargs = {
         'tiles': 'Stamen Toner',
         'zoom_start': 17,
         'width': 960,
         'height': 500,
-        'points': [],
-        'regions': []
+        'features': [],
     }
-    map_html = Map(location=center, **kwargs).map().to_html()
-    assert isinstance(map_html, HTML)
+    Map(**kwargs).show()
 
 
-def test_draw_map_arg(region1):
-    """ Tests that draw_map accepts a MapRegion or MapPoint as the first arg """
-    map_html = draw_map(region1)
-    assert isinstance(map_html, HTML)
+def test_map_marker():
+    """ Tests that a Map can contain a Marker. """
+    marker = Marker(51.514, -0.132)
+    Map([marker]).show()
 
-    point = MapPoint((51.5135015, -0.1358392))
-    map_html = draw_map(point)
-    assert isinstance(map_html, HTML)
 
+def test_map_region(states):
+    """ Tests that a Map can contain a Region. """
+    region = states['CA']
+    Map([region]).show()
```

Comment:
This is really neat; could `Map` accept region/marker as a positional argument? i.e., wrap the `features` arg as a tuple if a subclass of `_FoliumWrapper`.